### ISSUE TITLE
alexandria+ not found issue

### ISSUE THREAD

**Issue by jgarte** [Lisp-Stat/numerical-utilities#10]
```
Invoking sbcl: "/gnu/store/j2ylarkki92agq5i1apsh3c5y62hmznd-sbcl-2.2.6/bin/sbcl" "--non-interactive" "--eval" "(require :asdf)" "--eval" "(asdf:initialize-sourc
e-registry (list :source-registry (list :tree (uiop:ensure-pathname \"/gnu/store/3mkpc91h5jr412xk2hx92zx2db6w1m3d-sbcl-numerical-utilities-1.2.0-0.c1c308d/share
/common-lisp/sbcl/numerical-utilities\" :truenamize t :ensure-directory t)) :inherit-configuration))" "--eval" "(asdf:load-system \"num-utils\")" "--eval" "(asd
f:load-system \"num-utils/tests\")" 
This is SBCL 2.2.6, an implementation of ANSI Common Lisp.
More information about SBCL is available at <http://www.sbcl.org/>.

SBCL is free software, provided as is, with absolutely no warranty.
It is mostly in the public domain; some portions are provided under
BSD-style licenses.  See the CREDITS and COPYING files in the
distribution for more information.
; compiling file "/gnu/store/3mkpc91h5jr412xk2hx92zx2db6w1m3d-sbcl-numerical-utilities-1.2.0-0.c1c308d/share/common-lisp/sbcl/numerical-utilities/src/packages.l
isp" (written 01 JAN 1970 12:00:00 AM):
Unhandled PACKAGE-DOES-NOT-EXIST in thread #<SB-THREAD:THREAD "main thread" RUNNING
                                              {1001980003}>:
  The name "ALEXANDRIA+" does not designate any package.

Backtrace for: #<SB-THREAD:THREAD "main thread" RUNNING {1001980003}>
```

**Comment by jgarte** [comment]
Is there a way to structure this package differently so that this workaround is not needed?

https://github.com/Lisp-Stat/numerical-utilities#symbol-conflicts-with-alexandria

**Comment by jgarte** [comment]
I think that this can be resolved with Package-local nicknames in Common Lisp:

https://gist.github.com/phoe/2b63f33a2a4727a437403eceb7a6b4a3

**Comment by jgarte** [comment]
This is a bug:

https://github.com/Lisp-Stat/numerical-utilities/blob/master/src/packages.lisp#L38

because alexandria is not mentioned in https://github.com/Lisp-Stat/numerical-utilities/blob/master/num-utils.asd

**Comment by snunez1** [comment]
Closed with PR #11
